### PR TITLE
Fix CreateNewIndicatorsOnly script

### PR DESCRIPTION
**Pull request by DinaMeylakh**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-35607)

## Description
Added a flag to the command to indicate if association to incident is needed & a retry mechanism if it is.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by DinaMeylakh** [comment]
TDD: failing ut [here](https://gitlab.xdr.pan.local/xdr/cortex-content/content/-/jobs/5545960) at commit [db63cb7](https://github.com/demisto/content/pull/33907/commits/db63cb7a38d9f8c2f44d36909bb51734aff6feee)

File: Packs/CommonScripts/ReleaseNotes/1_14_38.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### CreateNewIndicatorsOnly
+
+- Added the `associate_to_current` argument and set its default behavior to false.
```
Comment (ShahafBenYakir) [inline Packs/CommonScripts/ReleaseNotes/1_14_38.md:6]:
```suggestion
- Added the `associate_to_current` argument which associates the indicator with the current incident.
```

File: Packs/CommonScripts/Scripts/CreateNewIndicatorsOnly/CreateNewIndicatorsOnly.py [hunk 2]
```diff
@@ -9,22 +10,41 @@
...
+        except Exception as err:
+            if "For associateIndicatorToIncident found no indicator" not in str(err):
+                raise err
+            # else log and continue with retries after sleeping
+            demisto.debug(f"Failed to find indicator {indicator_value} in the system.")
+            if retry_num != MAX_FIND_INDICATOR_RETRIES:
+                time.sleep(SLEEP_TIME)  # pylint: disable=E9003
+            retry_num += 1
+
+    if res != 'done':
```
Comment (ShahafBenYakir) [inline Packs/CommonScripts/Scripts/CreateNewIndicatorsOnly/CreateNewIndicatorsOnly.py:47]: execute command should return an entry no? I see that your code looks for a string

**Review by ShahafBenYakir — COMMENTED** [review]
Nice

File: Packs/CommonScripts/Scripts/CreateNewIndicatorsOnly/CreateNewIndicatorsOnly.py (hunk 2 above)
Comment (DinaMeylakh) [reply Packs/CommonScripts/Scripts/CreateNewIndicatorsOnly/CreateNewIndicatorsOnly.py:47]: you're thinking of `demisto.executeCommand`
`execute_command` on the other hand fails on error and returns the value of the command (in this case the string 'done' on success)

Comment (ShahafBenYakir) [reply Packs/CommonScripts/Scripts/CreateNewIndicatorsOnly/CreateNewIndicatorsOnly.py:47]: nice

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**